python/dgl/transform.py

```diff
@@ -2902,27 +2902,36 @@ def sort_csc_by_tag(g, tag, tag_offset_name='_TAG_OFFSET'):
     return new_g
 
 
-def reorder(g, permute_algo='rcmk', store_ids=True, permute_config=None):
-    r"""Return a new graph which re-order and re-label the nodes
+def reorder_graph(g, node_permute_algo='rcmk', edge_permute_algo='src',
+                  store_ids=True, permute_config=None):
+    r"""Return a new graph with nodes and edges are re-ordered/re-labeled
     according to the specified permute algorithm.
 
     Support homogeneous graph only for the moment.
 
-    This API is basically implemented by leveraging :func:`~dgl.node_subgraph`,
-    so the function signature is similar and raw IDs could be stored
-    in ``dgl.NID`` and ``dgl.EID``.
+    This API is basically implemented by leveraging :func:`~dgl.node_subgraph`
+    and :func:`~dgl.edge_subgraph`, so the function signature is similar and
+    raw IDs could be stored in ``dgl.NID`` and ``dgl.EID`` via :attr:`store_ids`.
```

Comment:
Move this paragraph to the end since it is about implementation details. I actually prefer to delete it.